src/main/java/org/jenkinsci/plugins/registry/notification/webhook/dockertrustedregistry/DockerTrustedRegistryPushNotification.java

```diff
@@ -65,10 +66,6 @@ public String getRegistryHost() {
         return registryHost;
     }
 
-    public void setRegistryHost(String registryHost) {
```

Comment:
what about `setImageTag` and `setImageDigest`?

Reply:
will remove those